src/assets/scss/FloatingActionButton.scss

```diff
@@ -0,0 +1,49 @@
+.parentfab {
+  position: relative
+}
+
+.fab {
+  width: 7.0rem;
+  height: 7.0rem;
```

Comment:
Add `box-shadow` on the fab. `box-shadow: 0.2rem 0.2rem 1rem rgba(0, 0, 0, 0.5)`

Reply:
Done @john555